client/src/Components/App/App.js

```diff
@@ -22,6 +23,7 @@ class App extends Component {
         <Route exact path="/contactUs" component={Contact} />
         <Route exact path="/login" component={Login} />
         <Route exact path="/register" component={Register} />
+        <Route exact path="/GenerateStaffLogin" component = {GenerateStaffLogin} />
```

Comment:
LOL, this is really tedious but just for consistency - can you please change it to /generateStaffLogin?
Thank you :)